install/updates/froxlor/0.10/update_0.10.inc.php

```diff
@@ -263,3 +263,9 @@
 if (\Froxlor\Froxlor::isFroxlorVersion('0.10.0-rc1')) {
 	\Froxlor\Froxlor::updateToVersion('0.10.0-rc2');
 }
+
+if (\Froxlor\Froxlor::isDatabaseVersion('201904250')) {
+
+	Database::query("ALTER TABLE `" . TABLE_PANEL_DOMAINS . "` ADD `caa` text default NULL AFTER `letsencrypt`;");
+	\Froxlor\Froxlor::updateToDbVersion('201907270');
```

Comment:
if you update the D version, please also update the value in https://github.com/Froxlor/Froxlor/blob/master/install/froxlor.sql for new installations and https://github.com/Froxlor/Froxlor/blob/master/lib/Froxlor/Froxlor.php#L13 for the system to know at what db version it is

Reply:
fixed in 5eef98fdfd0d3814b1796f1f72628d0b17cdab50.